docs/pages/develop/development-builds/introduction.mdx

```diff
@@ -7,8 +7,6 @@ searchPosition: 1
 ---
```

Comment:
```suggestion
hideTOC: true
---
```

Let's hide the table of contents on this page since there's only one section left.

![CleanShot 2025-06-04 at 00 47 54](https://github.com/user-attachments/assets/475c6db7-3d31-4a7d-ab28-a0259f18af77)